app/code/Magento/Ui/view/base/web/js/form/form.js

```diff
@@ -277,7 +277,9 @@ define([
             var invalidField = _.find(this.delegate('checkInvalid'));
 
             if (!_.isUndefined(invalidField) && _.isFunction(invalidField.focused)) {
-                invalidField.focused(true);
+                setTimeout(function(){
```

Comment:
Is there a way to resolve the issue without `setTimeout`. This solution is not reliable in all circumstances, we never know if timeout of 100 will be enough for all situations.